contents/install/update/major/0.7-to-0.8.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Update from 0.7 to 0.8-pre
+subtitle: Unstable development version!
+target_version: 0.8.0.0
+---
+
+<%= warning_box_start("danger") %>
+diaspora\* 0.8.0.0 is not released yet, this is about running the `develop` branch at the moment. You should not do that if you don't know exactly what you're doing
+<%= warning_box_end %>
+
+You need to install another package, `libidn11-dev` for debian and ubuntu, to run the current develop version of diaspora*.
```

Comment:
```suggestion
You need to install another package, `libidn11-dev` for Debian and Ubuntu, to run the current development version of diaspora*.
```

What about RPM based distros?

Reply:
This is copied from the Wiki, and since 0.8.0.0 is not released yet, that probably doesn't matter. There's a lot of stuff missing here, and I'm sure this will be expanded and polished once 0.8.0.0 is released.

The note in question was [added by Fla](https://wiki.diasporafoundation.org/wiki/index.php?title=Updating&type=revision&diff=10292&oldid=10289) after a support case, but I think we'll be doing proper testing pre-release anyway.